docs/index.md

```diff
@@ -0,0 +1,100 @@
+# Open Food Facts client for your Python applications and scripts
```

Comment:
This should be a symlink to ../README.md so that updates are automatic.

Reply:
I don't think symlinks get rendered in the website. I instead copy README.md each time, before mkdocs website generation.